README.md

```diff
@@ -439,6 +439,35 @@ Make sure you've completed the following assignments:
+
+Fork this repository on GitHub. You do that by clicking on `Fork`:<br/>
+![image](https://user-images.githubusercontent.com/84760072/219558561-441e6462-035c-4807-932b-bff465e98968.png)<br/><br/>
+Name your fork something informative by changing the text under `Repository name` and then click on `Create fork`:
+![image](https://user-images.githubusercontent.com/84760072/219558663-8f2a6a63-fa10-428b-99a0-ffb316b3a521.png)<br/><br/>
+After forking the repository, you will be in your forked repository. Which looks something like this:<br/>
+![image](https://user-images.githubusercontent.com/84760072/219558813-56130098-7f28-4b18-9d60-5f29d01fd1a0.png)<br/><br/>
+Make sure your forked repository is in sync with the upstream branch. You will have to do this periodically to ensure that your fork is up to date. You do that by clickin on `Sync fork`:<br/>
+![image](https://user-images.githubusercontent.com/84760072/219559314-5b2da1ee-2020-44f5-9555-8d0d345e3f36.png)<br/><br/>
+If your fork is out of date, this box will appear. Compare the changes by clicking on `Compare`. Once you are satisfied with the changes, click on `Update Branch`:<br/>
+![image](https://user-images.githubusercontent.com/84760072/219559801-46ab912b-5910-4e61-b5e1-9885b963ace1.png)<br/><br/>
+You don't want to make commits straight to your main branch so make a new branch. You do that by clicking on `[number] branch`. [number] can be any number, all that matters is you click on it.<br/>
+![image](https://user-images.githubusercontent.com/84760072/219560160-b87d5dfc-aa55-4051-bd7d-ab8add748190.png)<br/><br/>
+You will be redirected to this page. You then create a new branch by clicking on `New branch`<br/>
+![image](https://user-images.githubusercontent.com/84760072/219560296-4493f20d-b791-4266-b79a-d34cdd7d3841.png)<br/><br/>
+This modal will pop up. Give your branch an informative name under `Branch name` and then click on `Create branch`<br/>
+![image](https://user-images.githubusercontent.com/84760072/219560979-2db03899-15af-4aa9-b299-bfd022d850eb.png)<br/><br/>
+Make whatever changes 🔨 you want to make, then commit and push them to your fork. Once you are satisfied with your changes, you should make a pull request! You do that by clicking on `Contribute`<br/>
+![image](https://user-images.githubusercontent.com/84760072/219561275-2ae65aa1-50d2-4bb3-83fc-85b6e61d2952.png)<br/><br/>
+A box will appear, click on `Open pull request`<br/>
+![image](https://user-images.githubusercontent.com/84760072/219561409-a82d6477-aae9-4908-8086-be050da0fc21.png)<br/><br/>
+Give your pull request a descriptive title and a useful description. Once you are ready, click on `Create pull request`<br/>
+![image](https://user-images.githubusercontent.com/84760072/219561698-a82d16bc-d6d6-4783-8222-e24c14239d04.png)<br/><br/>
+The owner of the repository will later give you feedback on your pull request. Respond to it and revise your changes based on feedback.<br/>
```

Comment:
replace owner with maintainer(s)

promising feedback on the pull request may be a bit much. Sometimes there will be feedback, sometimes the PR gets ignored :see_no_evil: 

maintainers are busy people too and sometimes you need to be patient, and sometimes bring more attention to something

not sure how to best fix this besides saying something like

"The maintainer(s) of this repository will now be able to review your changes and provide feedback and guidance on your proposed contributions. You may need to add more commits or close and open a new pull request based on their feedback. Good luck!"